### PR TITLE
Responsive pages

### PR DESCRIPTION
**Pull request by ibtisam-adil**
### **Responsive**

-  Made pages responsive according to mobile screen and desktop.
- Created Mobile menu.
-  Added overlay on mobile menu.
- Added README file
- Deployed project, added link in README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,11 +3,187 @@
...
+            <img src="./images/img/mBlack.png"/>
+            <i class='bx bx-menu' id="menu-icon"></i>
+            <div class="nav-links hidden">
+                <a href="./about.html">About</a>
+                <a>Program</a>
+                <a>Join</a>
+                <a>Faqs</a>
+                <a>News</a>
+                <a>Login</a>
+            </div>
```
Comment (muneeb3123) [inline index.html:30]: The links on your page are extending beyond the container width, affecting the layout. To fix this, apply proper sizing to ensure they stay within bounds.

ibt-res

File: index.html [hunk 2]
```diff
@@ -3,11 +3,187 @@
...
+                <p class="desc-style">Zia-ul-Haq took power in 1977, imposed martial law and Islamic laws, curtailed civil liberties. He died in 1988.</p>
+            </div>
+            <div class="box">
+                <img src="./images/img/Icon-1.svg">
+                <p class="date-col">2008</p>
+                <p class="desc-style">Musharraf's rule saw economic and social reforms, better ties with West, but marred by corruption, abuses. Declared emergency in 2007, resigned in 2008.</p>
+            </div>
+        </div>
+        <a class="detail-link">See the whole program</a>
+    </section>
```
Comment (muneeb3123) [inline index.html:86]: Please follow the original design
please see 👇
Your-design


how they should be 👇
Orignal-design

File: index.html (hunk 1 above)
Comment (muneeb3123) [inline index.html:30]: Kindly fix the following error 👇


https://github.com/ibtisam-adil/Capstone/assets/118625824/22a9281a-ff56-4603-9a2e-b05d7f7c21ed

File: style.css [hunk 3]
```diff
@@ -1,8 +1,408 @@
...
+  font-family: "lato", sans-serif;
+}
+
+html {
+  line-height: 1.15;
+  -webkit-text-size-adjust: 100%;
+  font-size: 62.5%;
+  overflow-x: hidden;
+}
+
```
Comment (muneeb3123) [inline style.css:18]: I noticed that while your sections are looking good, there seems to be a slight alignment issue, Please fix it

alignment

File: index.html [hunk 4]
```diff
@@ -3,11 +3,187 @@
...
+
+        <!-- RULER'S SECTION -->
+    <section class="pak-rulers">
+        <div class="main-heading r-heading">
+            <p>Pakistan's Rulers</p>
+            <div class="line"></div>
+        </div>
+        <div class="all-leaders">
```
Comment (muneeb3123) [inline index.html:94]: [OPTIONAL]
Please follow the original design see 👇

ruler2

how it should be 👇
rulers-1

**Review by muneeb3123 — CHANGES_REQUESTED** [review]
Hi @ibtisam-adil  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark:
github flow :heavy_check_mark: 
follows orignal design :heavy_check_mark: 
media queries :heavy_check_mark: 
responsive :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: index.html (hunk 1 above)
Comment (ibtisam-adil) [reply index.html:30]: Fixed

File: index.html (hunk 2 above)
Comment (ibtisam-adil) [reply index.html:86]: fixed

File: index.html (hunk 1 above)
Comment (ibtisam-adil) [reply index.html:30]: fixed

File: style.css (hunk 3 above)
Comment (ibtisam-adil) [reply style.css:18]: fixed

File: style.css [hunk 5]
```diff
@@ -1,8 +1,409 @@
...
```
Comment (muneeb3123) [inline style.css:18]: You still need to fix it. please see the comparison in following

https://github.com/ibtisam-adil/Capstone/assets/118625824/538a3038-27fa-4100-9174-5466d97a13c4

**Review by muneeb3123 — CHANGES_REQUESTED** [review]
Hi @ibtisam-adil  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark:
github flow :heavy_check_mark: 
follows orignal design :heavy_check_mark: 
media queries :heavy_check_mark: 
responsive :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by muneeb3123 — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @ibtisam-adil , 
I'm Muneeb.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

-----

**Pull request merged**